docset/winserver2022-ps/networkcontrollerdiagnostics/Get-NetworkControllerManagedDevices.md

```diff
@@ -0,0 +1,111 @@
+---
+description: The Get-NetworkControllerManagedDevices cmdlet gets managed devices for a Network Controller.
+external help file: Debug-NetworkController.psm1-help.xml
+Module Name: NetworkControllerDiagnostics
+ms.date: 10/22/2021
+online version: https://docs.microsoft.com/powershell/module/networkcontrollerdiagnostics/get-networkcontrollermanageddevices?view=windowsserver2022-ps&wt.mc_id=ps-gethelp
+schema: 2.0.0
+title: Get-NetworkControllerManagedDevices
+---
+
+# Get-NetworkControllerManagedDevices
+
+## SYNOPSIS
+Gets managed devices for a Network Controller.
+
+## SYNTAX
+
+```
+Get-NetworkControllerManagedDevices [-RestURI] <String> [[-Credential] <PSCredential>]
+ [[-CertificateThumbprint] <String>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Get-NetworkControllerManagedDevices** cmdlet gets managed devices for a Network Controller. The cmdlet aggregates results for devices from **Get-NetworkControllerServer**, **Get-NetworkControllerVirtualServer**, **Get-NetworkControllerGateway**, and **Get-NetworkControllerLoadBalancerMux**, and then presents the connections information for every device.
+
+## EXAMPLES
+
+### Example 1: Get managed devices
+```powershell
+Get-NetworkControllerManagedDevices -RestURI  "https://nc.contoso.cloud.com" 
```

Comment:
Looks like there is an extra space between -RestURI and the URI. Is that intentional?

Reply:
Nope - I'll fix it